roles/openshift_control_plane/templates/master-controllers.j2

```diff
@@ -0,0 +1,65 @@
+#! /bin/bash
```

Comment:
One script template should be sufficient

Reply:
I think, it is better to have separate scripts because we any way need to  have one for cri-o and one for docker, without that combination becomes difficult to maintain, for example having a single script file for etcd, master-api and master-controller with changes for docker and crio will be hard to maintain and may lead to confusion.

Reply:
Well, we could still have one template for docker and CRI-O, but not for particular service we're deploying

Reply:
Ok, so you meant  one  template for all services(master-controller-service.j2) not for scripts(master-controllers.j2)